sdks/java/extensions/protobuf/src/main/java/org/apache/beam/sdk/extensions/protobuf/ProtoSchemaTranslator.java

```diff
@@ -156,6 +165,16 @@ static Schema getSchema(Class<? extends Message> clazz) {
   }
 
   static Schema getSchema(Descriptors.Descriptor descriptor) {
```

Comment:
Sorry, missed this: The method is now no longer thread-safe due to the use of a static hash map. I think that's acceptable for a package-private method, but this should be added to the Javadoc in case anyone in the future tries to use this in a multithreaded context.

Reply:
Resolving by marking the method as synchronized, and removing the javadoc